WebRoot/messages/ZMsg.properties

```diff
@@ -37,6 +37,7 @@ account.TOO_MANY_IDENTITIES = The identity could not be created because you have
 account.PASSWORD_CHANGE_TOO_SOON = Password cannot be changed too soon.
 account.PASSWORD_LOCKED = Password locked. User not allowed to change password. Please contact your System Administrator.
 account.PASSWORD_RECENTLY_USED = The password you submitted has recently been used. Please choose a different password.
+account.WEB_CLIENT_ACCESS_NOT_ALLOWED = Web client access is not allowed.
```

Comment:
please add translation for all languages

Reply:
Fixed.